Misc/NEWS.d/next/Documentation/2020-06-12-21-23-23.bpo-40896.mIMSEP.rst

```diff
@@ -0,0 +1 @@
+Add links to Library source code to Documentation. Enhanced by Edison Abahurire.
```

Comment:
Documentation only changes don't need a blurb.

Reply:
But they can and for non-trivial changes ofter do.  'Documentation' is a section in the blurb template and in the changelog.
https://docs.python.org/3/whatsnew/changelog.html#changelog
I would change and expand to
"Add source links to doc for 2to3, asyncio/*, ..., curses/*, ... wsgiref."

Reply:
I think in this case it could be reasonable either way, but I wouldn't consider it to be strictly required for small documentation enhancements.